core-bundle/contao/models/PageModel.php

```diff
@@ -927,6 +927,9 @@ public function loadDetails()
 		$trail = array($this->id, $pid);
 		$time = time();
 
+		// Backwards compatibility fallback (see #8252)
+		$this->noSearch = 'never_index' === $this->searchIndexer ? true : false;
+
```

Comment:
As discussed in the Contao call, the fallback should be moved to the `__get()` method instead.

Reply:
> I couldn't find the correct place.

Here in the `PageModel` class you will need to do something like this:

```php
/**
 * Return an object property
 *
 * @param string $strKey The property key
 *
 * @return mixed|null The property value or null
 */
public function __get($strKey)
{
	if ('noSearch' === $strKey)
	{
		return 'never_index' === $this->searchIndexer;
	}

	return parent::__get($strKey);
}
```